server/controllers/article_controller.js

```diff
@@ -1,21 +1,27 @@
 import Article from '../models/article';
-import * as Group from './group_controller';
+import * as Groups from './group_controller';
 
-export const createArticle = (uri, groupid) => {
+// Should only be called when creating annotations
+export const createArticle = (uri, groupIds) => {
   const article = new Article();
   article.uri = uri;
-  article.groups.push(groupid);
-  console.log(groupid);
+  article.groups = groupIds;
   return article.save()
-      .then(result => {
-        Group.addGroupArticle(groupid, article._id);
-      });
+  .then(result => {
+    Groups.addGroupArticle(groupIds, result._id);
```

Comment:
If we get a promise from this, should chain on a .then and put the return statement inside. Can still return the same "result" from above.